examples/experiment-1.0.amp.html

```diff
@@ -10,6 +10,11 @@
   <script async custom-element="amp-user-notification" src="https://cdn.ampproject.org/v0/amp-user-notification-0.1.js"></script>
   <script async src="https://cdn.ampproject.org/v0.js"></script>
   <style amp-custom>
+    amp-user-notification {
```

Comment:
What's the change for?

Reply:
This is for the example, to add a background to the "allow background color" setting for the experiment. It was just hard to see so I added that 😄